packages/zata/test/Adaptor.test.js

```diff
@@ -0,0 +1,112 @@
+import { expect } from 'chai';
+import { enableMockClient } from '@openfn/language-common/util';
+import { request, get, post, put } from '../src/Adaptor.js';
+import testData from './fixtures.json' assert { type: 'json' };
+
+const testServer = enableMockClient('https://fake.server.com');
```

Comment:
Can we set  this URL to the default zata, so that the tests look a bit more realistic?